### DISCUSSION TITLE
Welcome to Healthchecks Discussions!

### DISCUSSION THREAD

**Discussion by cuu508** [healthchecks/healthchecks#459] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hansaliyad1** [comment]
Is healthcheck.io more than just health check? Can I write some functions in JS which can create PDFs, pull records from DSN source using node-odbc library etc?

**Reply by cuu508** [reply]
Healthchecks.io is primarily a cron job monitoring service – https://healthchecks.io/docs/monitoring_cron_jobs/
It is open source, the code is in this repository, and you can self-host it.
